doc/classes/ItemList.xml

```diff
@@ -89,6 +91,7 @@
 			<argument index="0" name="idx" type="int">
 			</argument>
 			<description>
+				Returns the the icon assosiated with [code]idx[/code] item. Default value is [code]null[/code]
```

Comment:
Typo, "associated" with a C. I would also say "with *the* [code]idx[/code] item". And end the second sentence with a dot.

Reply:
> I would also say "with *the* [code]idx[/code] item"

Alternatively, "with item [code]idx[/code]". I'm not a native speaker so I'm not sure which is best (cc @cbscribe).

Reply:
Installed spell checking for visual studio code!